pkg/kubectl/cmd/replace.go

```diff
@@ -200,17 +210,37 @@ func forceReplace(f cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []s
 	if err != nil {
 		return err
 	}
+
 	r := resource.NewBuilder(mapper, typer, resource.ClientMapperFunc(f.UnstructuredClientForMapping), unstructured.UnstructuredJSONScheme).
 		ContinueOnError().
 		NamespaceParam(cmdNamespace).DefaultNamespace().
 		FilenameParam(enforceNamespace, options).
 		ResourceTypeOrNameArgs(false, args...).RequireObject(false).
+		Latest().
```

Comment:
> It means force replacing an object that doesn't exist will fail (it works today because a NotFound error is tolerated), it potentially means a double fetch,

Can you clarify this? Is it going to have some other types of errors?

> I'm not sure preserving the annotation from an object so different we had to delete it is the right thing to do.

What should we do here? WDYT?